scripts/minikube-setup.sh

```diff
@@ -15,6 +15,24 @@
 
 set -ex
 
-curl -Lo minikube https://storage.googleapis.com/minikube-builds/master/minikube-linux-amd64
-sudo install minikube /usr/local/bin/minikube
-minikube start --profile=minikube --driver=docker
\ No newline at end of file
+# conntrack is required for minikube 1.19 and higher for none driver
+if ! conntrack --version &>/dev/null; then
+  echo "WARNING: No contrack is not installed"
+  sudo apt-get update -qq
+  sudo apt-get -qq -y install conntrack
+fi
+
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
```

Comment:
TODO: Don't depend on latest minikube release?

Reply:
This is what it did before and is already strictly more conservative than `minikube-builds` from `master` 😲

Reply:
Oh absolutely, just something to clean up further in the future.

Reply:
Opened: https://github.com/GoogleContainerTools/kaniko/issues/1843